### DISCUSSION TITLE
Is this project still active?

### DISCUSSION THREAD

**Discussion by sacbhale** [TD22057/insteon-mqtt#509] [General]
I have been using this project for a couple of years now. Works great. Thank you to the developers.
Insteon the company has gone through a death and re-birth and seem to be starting to bring out updated products.
I have had insteon hardware for the last 10 years and would like to continue to use it with home assistant. 
Wondering if others are doing the same or people are moving to other platforms.
It has been a long time since the last merge.
Mostly I want to know if this project will keep getting updates with new products and to keep up with changes that home assistant might make to discovery etc.

**Comment by EnGamma** [comment]
@sacbhale 

I am an extensive user of Insteon devices and a recent adopter of insteon-mqtt, which I'm currently using outside of HomeAssistant, but have plans to use it with HomeAssistant in the near future when I can free up some time.    I also have some ideas for enhancements based on my experience thus far will be looking into contributing later this year.   I also have an interest in expanding the types of supported devices beyond those that are supported today.

**Comment by krkeegan** [comment]
The project is still alive, but very stable at this point.

Ted started the project and is still around, but my understanding is he doesn't use Insteon products anymore.  I contributed a fair amount to get things working for me and I still use it daily and extensively (~100 Insteon devices in my house), but it already just works for me, I don't have any thing I would add at this point.  

I think there are a few maintenance patches that I made that still need to be merged into master.  Also, someone did a lot of work on improving the configuration of the home assistant discovery function.  That has been sitting around for a long time, and I probably just need to merge it at this point.

In terms of adding new devices, this is going to be on the shoulders of whoever wants the new device.  It is kinda hard to do the coding if you don't own the device.  That said, I am happy to merge commits that add new devices.  And if I happen to buy a new device, I will be happy to work on the code for that.

Similarly, if there are features that users want, I am happy to merge the commits, but I do not plan on taking an active role in developing new features as again, everything just works for me.

On the upshot, the ecosystem is very stable.  Outside of Home Assistant, the MQTT and Insteon environment are not changing, so this project should continue to work without needing any updates for a very long time.

The only aspect that seems to change is our limited interaction with Home Assistant, but even here, most of the aspects that we use are very stable.  Small adjustments and tweaks may be necessary as Home Assistant develops more, but I don't expect them to be much.

I hope that answers everyone's questions.

_edit_ Missed a "do not" when discussing new features.
